library/xwiki

```diff
@@ -1,18 +1,18 @@
 Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
-Tags: 9, 9.11, 9.11.2, 9-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts, latest
+Tags: 9, 9.11, 9.11.3, 9-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts
 GitCommit: bdf1a8464a668061ba564249461633f0445e08a6
 Directory: 9/mysql-tomcat
 
-Tags: 9-postgres-tomcat, 9.11-postgres-tomcat, 9.11.2-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
+Tags: 9-postgres-tomcat, 9.11-postgres-tomcat, 9.11.3-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
 GitCommit: bdf1a8464a668061ba564249461633f0445e08a6
 Directory: 9/postgres-tomcat
 
-Tags: 10, 10.1, 10-mysql-tomcat, 10.1-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
+Tags: 10, 10.1, 10-mysql-tomcat, 10.1-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
 GitCommit: ac81dad2ed329469be2ed8b624fd2bc7a44455c4
 Directory: 10/mysql-tomcat
 
-Tags: 10-postgres-tomcat, 10.1-postgres-tomcat, stable-postgres-tomcat, stable-postgres
+Tags: 10-postgres-tomcat, 10.1-postgres-tomcat, stable-postgres-tomcat, stable-postgres, latest-postgres
```

Comment:
See https://github.com/docker-library/official-images#tags-and-aliases (and https://github.com/docker-library/official-images/pull/4143 :innocent:). :heart: